script.js

```diff
@@ -0,0 +1,89 @@
+// Side mobile menu
+const openMenu = document.querySelector('.sideMenu');
+const headP = document.querySelector('.headParaDiv');
+const mobileNav = document.querySelector('.mobileNav');
+
+function openMobileMenu() {
+  mobileNav.style.display = 'block';
+  openMenu.style.display = 'none';
+  headP.style.display = 'none';
+}
+openMobileMenu();
+
+function closeMobileMenu() {
+  mobileNav.style.display = 'none';
+  openMenu.style.display = 'block';
+  headP.style.display = 'block';
+}
+closeMobileMenu();
+
+// team Object
+const team = [{
+  img: 'img/avatar1.png',
+  desc: ['Ayebazibwe Ishmael', 'Full stackWeb developer', '<hr>', 'Ishmael is CEO My plug, who has experience in innovating through technology and sftware, for he has enabled the plug store to function'],
+},
+{
+  img: 'img/avatar2.jpg',
+  desc: ['Natweta Edmon', 'Grapphics Designer', '<hr>', 'Edmond heads the graphic department, He is responisible for the brand look of our clients that are need of that service'],
+},
+{
+  img: 'img/avatar5.jpg',
+  desc: ['Masaba Johhn', 'Producer', '<hr>', 'John is the best producer around town, for all our  clients production are readily produced by John Masaba the man'],
+},
+{
+  img: 'img/avatar3.jpg',
+  desc: ['Apio Sheillah', 'Video Director', '<hr>', 'Sheillah is in charge of video and audio commercials for our clients that run on both television and radios plus social media'],
+},
+{
+  img: 'img/avatar4.jpg',
+  desc: ['Nazziwa Jane', 'Marketier', '<hr>', 'Jane dose not only market for My plug but also  my plug store creators who sell and and create products on my plug store'],
+},
+{
+  img: 'img/avatar6.jpg',
+  desc: ['Richard Nabende', 'Program cordinator', '<hr>', 'Each year we handle diffrent kinds of project for our clients, Richard makes sure that projects are managed effeciently.'],
+},
+];
```

Comment:
**Changes Suggested**
- [ ] Good effort on storing the teams in an array. However, I propose that you remove the `desc array` and have the `name, title, and description` as individual object properties. For instance, `name: 'Richard Nabende` `title: 'Program Coordinator'`. This will help reduce the Big O notation time complexity when you `iterate` team array using an iterator, in this case the for loop.

Reply:
Fixed